File: spring/src/main/java/com/example/shoppingCart/models/OrderDetail.java
package com.example.shoppingCart.models;

import lombok.Data;

import javax.persistence.*;

@Entity
@Data
public class OrderDetail {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    private int quantity;

    @ManyToOne
    private Item item;

    @ManyToOne
    private Orde_r order;

}

File: Submission/angular/shopping-cart/src/app/models/OrderDetail.ts
import { IItem } from './Item';
import { IOrder } from './Order';

export interface IOrderDetail {
    id: number;
    quantity: number;
    item: IItem;
    order: IOrder;
}

export class Order {
    id: number = 0;
    quantity: number;
    item: IItem;
    order: IOrder;
}